test/interpreter/function-mirr.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function MIRR', () => {
+  it('should return #NA! error with the wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=MIRR(1,1)'],
+      ['=MIRR(1,1,1,1)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqual(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should return correct value', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=MIRR(B1:C1,1,1)', 1, -1],
```

Comment:
I would avoid using the same value for completely different parameters :) 
Can I also ask for the following test? 
`['=MIRR(B1:F1,0.3,-0.1)',1,2,-3,-5,8]`
(This will also check negative rates)